### PR TITLE
Text area max length

### PR DESCRIPTION
**Pull request by heeboA**
Set maximum text length attribute for text area fields.

### CODE REVIEW

File: resources/less/hakija.less [hunk 1]
```diff
@@ -446,6 +446,7 @@ textarea.application__form-text-input {
 }
 .application__form-text-area {
   width: 610px;
+  resize: none;
```
Comment (starvinraven) [inline resources/less/hakija.less]: Do we want to do this? Seems like a usability downgrade (for some)...

File: src/cljs/ataru/hakija/application_form_components.cljs [hunk 2]
```diff
@@ -207,20 +207,27 @@
...
-        [info-text field-descriptor]]
-       [:textarea.application__form-text-input.application__form-text-area
-        {:id (:id field-descriptor)
-         :class (text-area-size->class (-> field-descriptor :params :size))
-         ; default-value because IE11 will "flicker" on input fields. This has side-effect of NOT showing any
-         ; dynamically made changes to the text-field value.
-         :default-value (textual-field-value field-descriptor @application)
-         :on-change on-change
-         :value (textual-field-value field-descriptor @application)}]])))
+      (println max-length)
```
Comment (starvinraven) [inline src/cljs/ataru/hakija/application_form_components.cljs]: Stray println here

File: src/cljs/ataru/hakija/application_form_components.cljs [hunk 3]
```diff
@@ -207,20 +207,27 @@
...
+          {:id            (:id field-descriptor)
+           :class         (text-area-size->class size)
+           :maxLength     max-length
+           ; default-value because IE11 will "flicker" on input fields. This has side-effect of NOT showing any
+           ; dynamically made changes to the text-field value.
+           :default-value value
+           :on-change     on-change
+           :value         value}]
+         [:span.application__form-textarea-max-length
+          {:dangerouslySetInnerHTML {:__html (str (count value) " / " max-length)}}]]))))
```
Comment (starvinraven) [inline src/cljs/ataru/hakija/application_form_components.cljs]: Why is `dangerouslySetInnerHTML` needed here?

File: src/cljs/ataru/hakija/application_form_components.cljs [hunk 4]
```diff
@@ -207,20 +207,27 @@
 (defn text-area [field-descriptor & {:keys [div-kwd] :or {div-kwd :div.application__form-field}}]
   (let [application (subscribe [:state-query [:application]])
         answers     (subscribe [:state-query [:application :answers]])
-        on-change   (partial textual-field-change field-descriptor)]
+        on-change   (partial textual-field-change field-descriptor)
+        size        (-> field-descriptor :params :size)
+        max-length  (-> field-descriptor :params :max-length (or 1000))]
```
Comment (starvinraven) [inline src/cljs/ataru/hakija/application_form_components.cljs]: Is 1000 characters a sane default max-length? Seems a bit low?

File: resources/less/hakija.less (hunk 1 above)
Comment (heeboA) [reply resources/less/hakija.less]: By Antti's request.

File: src/cljs/ataru/hakija/application_form_components.cljs (hunk 4 above)
Comment (heeboA) [reply src/cljs/ataru/hakija/application_form_components.cljs]: This was just a preliminary suggestion for default values of S/M/L boxes with 500/1000/1500 chars respectively. We'll adjust if needed.

File: src/cljs/ataru/hakija/application_form_components.cljs (hunk 3 above)
Comment (heeboA) [reply src/cljs/ataru/hakija/application_form_components.cljs]: Honestly i don't know :smile:

**Comment by heeboA** [comment]
Merged by hand.

**Pull request closed without merging**